server/db/connection.js

```diff
@@ -0,0 +1,13 @@
+const { Client } = require('pg');
+const dotenv = require('dotenv');
+
+const client = new Client({
+  user: process.env.USER,
+  port: process.env.PORT,
```

Comment:
If I am not wrong I think .env port is 3000 and db stays in different port.

Reply:
will change